src/hooks/useVaultReallocations.ts

```
@@ -0,0 +1,132 @@
+      // Group by hash as each hash represents one logical reallocation
+      const hashGroups: Record<string, ReallocateItem[]> = {};
+      
+      items.forEach(item => {
+        if (!hashGroups[item.hash]) {
+          hashGroups[item.hash] = [];
+        }
+        hashGroups[item.hash].push(item);
+      });
+      
+      // Convert groups to our desired format
+      return Object.entries(hashGroups).map(([hash, items]) => {
```

ESLint

[error] 'items' is already declared in the upper scope on line 93 column 13.

(@typescript-eslint/no-shadow)